q2_sample_classifier/classify.py

```diff
@@ -350,6 +350,52 @@ def summarize(output_dir: str, sample_estimator: Pipeline):
     _summarize_estimator(output_dir, sample_estimator)
 
 
+def heatmap(ctx, table, importance, metadata=None, feature_count=50,
+            importance_threshold=0, group_samples=True, normalize=True,
```

Comment:
```suggestion
            importance_threshold=0, group_samples=False, normalize=True,
```

Thoughts?

Reply:
lgtm, already made change in my local branch.